### PR TITLE
removed duplicate declaration of style property

### PR DESCRIPTION
**Pull request by manpenaloza**
style property on span of render function was a duplicate. removed it.

### CODE REVIEW

File: src/components/Day.js [hunk 1]
```diff
@@ -55,7 +55,6 @@ export default class Day extends Component {
   render() {
     return (
       <span
-        style={ this.props.style }
```
Comment (manpenaloza) [inline src/components/Day.js]: see line 66 to find the second declaration of this property

**Comment by cezarneaga** [comment]
Hi @manpenaloza,
could you fix the failing tests?
thanks for the PR!

Cezar

**Comment by nikgraf** [comment]
@manpenaloza seems like you removed both, shouldn't you remove only one?

**Comment by manpenaloza** [comment]
f... thx, i messed up some commits. Will close this one and send a new one.

**Pull request closed without merging**